README.md

```diff
@@ -30,6 +30,20 @@ MATTR Wallet SDK React Native / [Exports](modules.md)
 
 # Getting started
 
+## How to get access to MATTR Pi Wallet SDK
+To gain access to the MATTR Pi Wallet SDK, please follow these steps. 
+
+1. Request or download the ["Terms of Agreement"](Licence.pdf).
+2. Read the "Terms of Agreement", sign it, and return it to us. 
+3. Create an account at [NPMJS - Node Package Manager for JavaScript](https://www.npmjs.com).
+4. Ensure multi-factor authentication (MFA) is Configured on NPMJS Account.
+5. Create a personal access token [Create a personal access token](https://docs.npmjs.com/.creating-and-viewing-access-tokenshttps://docs.npmjs.com/creating-and-viewing-access-tokens).
+6. Supply the NPMJS (Node Package Manager for JavaScript) account name back to MATTR.
+7. MATTR will process the request and allow you to access the Wallet SDK if all the information is complete. 
```

Comment:
@kimattr change to " MATTR will process the request and provision access to the MATTR Pi Wallet SDK if approved."

Reply:
@dylan-paul Done. Changes are pushed.